app/livechat/client/views/app/tabbar/visitorForward.js

```diff
@@ -83,8 +108,9 @@ Template.visitorForward.events({
 		const [user] = instance.selectedAgents.get();
 		if (user) {
 			transferData.userId = user._id;
-		} else if (instance.find('#forwardDepartment').value) {
-			transferData.departmentId = instance.find('#forwardDepartment').value;
+		} else if (instance.selectedDepartments.get()) {
+			const [department] = instance.selectedDepartments.get();
+			transferData.departmentId = department._id;
```

Comment:
```suggestion
			transferData.departmentId = department && department._id;
```

Reply:
Should check if the departmentId or the userId was set before call the method below

Reply:
Done.